src/graph/test/DataTest.cpp

```diff
@@ -106,6 +106,26 @@ AssertionResult DataTest::prepareSchema() {
                                << " failed, error code "<< static_cast<int32_t>(code);
         }
     }
+    // Test same propName diff tyep in diff tags
+    {
+        cpp2::ExecutionResponse resp;
+        std::string cmd = "CREATE TAG employee(name int)";
+        auto code = client_->execute(cmd, resp);
+        if (cpp2::ErrorCode::SUCCEEDED != code) {
+            return TestError() << "Do cmd:" << cmd
+                               << " failed, error code "<< static_cast<int32_t>(code);
+        }
+    }
+    // Test same propName same type in diff tags
+    {
+        cpp2::ExecutionResponse resp;
+        std::string cmd = "CREATE TAG interest(name string)";
+        auto code = client_->execute(cmd, resp);
+        if (cpp2::ErrorCode::SUCCEEDED != code) {
+            return TestError() << "Do cmd:" << cmd
+                               << " failed, error code "<< static_cast<int32_t>(code);
+        }
+    }
```

Comment:
1. how about use for-loop to create? because I don't see any case to cover: add 10000000 tags/edge_types.
```
for ()

2. create tag xxx+i (name int, name2 string)

2. create tag xxy+i (name string, name2 int)
```